packages/kotti-ui/source/props.ts

```diff
@@ -1,29 +1,16 @@
+import { PropOptions, PropType } from '@vue/composition-api'
 import { cloneDeep } from 'lodash'
 import { z } from 'zod'
 
-/**
- * Easily validate vue props, improve error messages drastically
- *
- * This augments/replaces Vue’s prop.validator approach by using zod and therefore actually
- * prints errors to the console that can actually help developers rather than just say
- * "hey you failed the validator, good luck!" as vue would do by default.
- *
- * @example
- * sections: {
- * 	required: true,
- * 	type: Array,
- * 	validator: propValidator(KottiUserMenu.propsInternalSchema, 'sections'),
- * }
- */
-export const propValidator = <
-	SCHEMA extends z.ZodObject<z.ZodRawShape>,
-	KEY extends keyof z.infer<SCHEMA>,
->(
+const DEBUG_MAKE_PROPS = false as const // enable to print debug log
+const DEBUG_WALK_SHAPE = false as const // enable to print debug log
+
+const propValidator = <SCHEMA extends z.ZodTypeAny>(
```

Comment:
ok so previously the schema was `z.ZodObject<X>`
where X was `z.ZodRawShape`

type declaration for raw shape is that it's an object of values of type `ZodTypeAny`:  
```
export declare type ZodRawShape = {
    [k: string]: ZodTypeAny;
};
```

HOW?

I would have understood if 
1) previously schema was 
`z.ZodZodObject<z.ZodTypeAny>`
2) now it became
`z.ZodRawShape`

none of this makes sense

I kind thing it should be simply
`SCHEMA extends z.ZodRawShape`